### ISSUE TITLE
GitHub Actions

### ISSUE THREAD

**Issue by teners** [piterpy-meetup/c4p2n#18]
Should we discuss replacing Travis with Actions? On the Free GitHub plan we've got 2k Actions minutes per month and it seems like more than enough. We can get rid of an excess part of our infrastructure which always seems to me like a win situation if it costs nothing.

**Comment by ave2me** [comment]
I agree with that. Seems like now it won't be much pain moving to Actions.

**Comment by b0g3r** [comment]
Agree
